src/main/java/io/seqera/tower/cli/commands/global/WorkspaceRequiredOptions.java

```diff
@@ -13,8 +13,8 @@
 
 import picocli.CommandLine;
 
-public class WorkspaceOptions {
+public class WorkspaceRequiredOptions {
 
-    @CommandLine.Option(names = {"-w", "--workspace"}, description = "Workspace numeric identifier (TOWER_WORKSPACE_ID)", defaultValue = "${TOWER_WORKSPACE_ID}")
+    @CommandLine.Option(names = {"-w", "--workspace"}, description = WorkspaceOptionalOptions.DESCRIPTION, defaultValue = WorkspaceOptionalOptions.DEFAULT_VALUE, required = true)
```

Comment:
Could we try using the same convention as for `WorkspaceOptionalOptions` when it comes to `description` and `defaultValue`? (I meant, using constants)

Reply:
I don't get what do you mean. Here I'm using the `WorkspaceOptionalOptions` constants to keep both in sync.

Reply:
Sorry, you`re right. I focused on the missing constant and missed the static reference to them.